src/ThreeFourteen.AlphaVantage/Response/Metadata.cs

```diff
@@ -12,6 +12,8 @@ public Metadata(IDictionary<string, string> dictionary) : base(dictionary)
 
         public override string ToString()
         {
+            if ( Keys == null )
+               return string.Empty;
```

Comment:
There is no point returning empty string. I think throwing an exception with an informative message would be clearer.